git/PKGBUILD

```diff
@@ -45,17 +45,31 @@ replaces=('git-core')
 provides=('git-core')
 #options=('debug' '!strip')
 source=("${pkgname}-${pkgver}.tar.gz"::https://github.com/git/git/archive/v${pkgver}.tar.gz
+        git.keyring
```

Comment:
It's a public PGP keyring of git project. It doesn't needed in most cases, it might be used in case of importing some patches from source tree and to [verify commit](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work#Verifying-Tags) (a tag normally). Just included it for convenience in case someone would require it at some point.